docs(config): Describe maxAsyncSize and maxInitialSize (#2756)
webpack/webpack.js.org: src/content/plugins/split-chunks-plugin.md

T> `maxSize` takes higher priority than `maxInitialRequest/maxAsyncRequests`. Actual priority is `maxInitialRequest/maxAsyncRequests < maxSize < minSize`.

Note that setting the value for `maxSize` sets the value for both `maxAsyncSize` and `maxInitialSize`.

### `splitChunks.maxAsyncSize`

`number`

> Like `maxSize`, `maxAsyncSize` can be applied globally (`optimization.splitChunks.maxAsyncSize`), to cacheGroups (`optimization.splitChunks.cacheGroups[x].maxAsyncSize`), or to the fallback cache group (`optimization.splitChunks.fallbackCacheGroup.maxAsyncSize`).

Reviewer: ```suggestion
Like `maxSize`, `maxAsyncSize` can be applied globally (`splitChunks.maxAsyncSize`), to cacheGroups (`splitChunks.cacheGroups.{cacheGroup}.maxAsyncSize`), or to the fallback cache group (`splitChunks.fallbackCacheGroup.maxAsyncSize`).
```

Reviewer: We omit optimization further down the examples, also use `splitChunks.cacheGroups.{cacheGroup}` vs `splitChunks.cacheGroups[x]` for consistency

> The difference between `maxAsyncSize` and `maxSize` is that `maxAsyncSize` will only be applied to chunks that are loaded on-demand.

Reviewer: A bit shorter without 2x `that` version for consideration
```suggestion
The difference between `maxAsyncSize` and `maxSize` is that `maxAsyncSize` will only affect on-demand loading chunks.
```

### `splitChunks.maxInitialSize`

`number`

> Like `maxSize`, `maxInitialSize` can be applied globally (`optimization.splitChunks.maxInitialSize`), to cacheGroups (`optimization.splitChunks.cacheGroups[x].maxInitialSize`), or to the fallback cache group (`optimization.splitChunks.fallbackCacheGroup.maxInitialSize`).

Reviewer: ```suggestion
Like `maxSize`, `maxInitialSize` can be applied globally (`splitChunks.maxInitialSize`), to cacheGroups (`splitChunks.cacheGroups.{cacheGroup}.maxInitialSize`), or to the fallback cache group (`splitChunks.fallbackCacheGroup.maxInitialSize`).
```
same as above

> The difference between `maxInitialSize` and `maxSize` is that `maxInitialSize` will only be applied to chunks that are used for the initial load.

Reviewer: Another short for consideration
```suggestion
The difference between `maxInitialSize` and `maxSize` is that `maxInitialSize` will only affect initial load chunks.
```
